src/integTest/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanelIntegTest.kt

```diff
@@ -244,16 +254,16 @@ class SnykToolWindowPanelIntegTest : HeavyPlatformTestCase() {
         val snykError = SnykError(SnykToolWindowPanel.NO_OSS_FILES, project.basePath.toString())
         val snykErrorControl = SnykError("control", project.basePath.toString())
 
-        toolWindowPanel.snykScanListener.scanningOssError(snykErrorControl)
-        toolWindowPanel.snykScanListener.scanningOssError(snykError)
+        scanPublisher.scanningOssError(snykErrorControl)
```

Comment:
Why do we react/listen to messages on a publisher? This seems counter-intuitive to me - maybe something like eventHandler as a name would be better, and the event-handler method names would be sth like `onScanningOssError` if you don't want to separate publisher and consumer?

Reply:
Here we don't react, but send the event to publisher (to imitate end of OSS scan).
Then all subscribers will react accordingly (and we'll check state after).
PS here we now need to send event to the publisher (instead of hacky way to use toolWindowPanel.snykScanListener) cause not only ToolWindow (but also SnykCachedResults too) is processing that events now.